src/models/ProjectTagLower.js

```diff
@@ -0,0 +1,7 @@
+const ProjectTagLower = 
+    `CREATE OR REPLACE TRIGGER ProjectTagLower 
+    BEFORE INSERT OR UPDATE ON "ProjectTag"
+    FOR EACH ROW
+    EXECUTE FUNCTION LowerInsert(tag)`;
+
+module.exports = ProjectTagLower;
```

Comment:
Could this have been included in ProjectTag.js as a named export? It's fine though.